ed_sensor_integration/src/laser/laser_plugin.h

```diff
@@ -11,13 +11,17 @@
 #include <ros/subscriber.h>
 #include <ros/callback_queue.h>
 #include <tf2_ros/buffer.h>
-#include <tf2_ros/transform_listener.h>
```

Comment:
Why did you replace this include with a forward declaration? Just curious.

Reply:
This sometimes reduces compilation and linking times.